### DISCUSSION TITLE
Quest two problems

### DISCUSSION THREAD

**Discussion by Z3R0R4ND0M** [Okabintaro/SubmersedVR#139] [Q&A]
So I'm on the quest two, and I have steam VR fully set up, and It won't work, I HAVE put in -vrmode openvr into the launch options, but it still won't work, I have done everything I can, is it a problem with wireless? or is it something else?

**Comment by Ztoxicanime** [comment]
I was also having the same problem. What ended up being the issue for me was that the Vortex version of Tobey's BepInEx Pack for Subnautica was missing files that the GitHub version had. If you followed a tutorial that said to use Vortex, just get the GitHub version and paste it over the Vortex version. Then reinstall SubmersedVR, and you should have a working game. (and make sure to type in -vrmode openvr instead of copying and pasting that might help as well.)
